cranelift/codegen/src/machinst/isle.rs

```diff
@@ -804,6 +804,18 @@ macro_rules! isle_prelude_methods {
             self.lower_ctx.use_constant(data)
         }
 
+        #[inline]
+        fn emit_u128_le_const(&mut self, lower: u64, upper: u64) -> VCodeConstant {
+            let bytes = lower
+                .to_le_bytes()
+                .iter()
+                .chain(upper.to_le_bytes().iter())
+                .cloned()
```

Comment:
Absolutely unimportant nit: I like to use `copied` instead of `cloned` when I'm expecting it to be cheap, so the compiler can tell me if I've tried to do it to a type that isn't `Copy`.